mantle/kola/tests/ignition/kdump.go

```
@@ -0,0 +1,205 @@
+	for _, line := range strings.Split(strings.TrimSuffix(string(PrivKeyBuf), "\n"), "\n") {
+		padded = fmt.Sprintf("%s          %s\n", padded, line)
+	}
+
+	return SshServer {
+		Machine:    m,
+		MachineAddress:    address,
+	    SSHPort: port,
+		PubSSH: string(pubkeyBuf),
+		PrivSSH: padded,
+	}
+}
+
+func kdumpSSHTest(c cluster.TestCluster) {
+	ssh_host := setupSSHMachine(c)
+	butane := conf.Butane(fmt.Sprintf(`variant: fcos
+version: 1.5.0
+storage:
+  files:
+    - path: /root/.ssh/id_ssh_kdump.pub
+      mode: 0600
+      contents:
+        inline: |
+          %s
+    - path: /root/.ssh/id_ssh_kdump
+      mode: 0600
+      contents:
+        inline: |
+%s
+    - path: /root/.ssh/config
+      mode: 0644
+      overwrite: true
+      contents:
+        inline: |
+           Host %s
+               StrictHostKeyChecking no
+               Port %s
+    - path: /etc/kdump.conf
+      overwrite: true
+      contents:
+        inline: |
+          ssh core@%s
+          sshkey /root/.ssh/id_ssh_kdump
+          path /home/core/crash
+          core_collector makedumpfile -F -l --message-level 1 -d 31
+systemd:
+  units:
+    - name: kdump.service
+      enabled: true
+kernel_arguments:
+    should_exist:
+      - crashkernel=512M`,
+	   ssh_host.PubSSH, ssh_host.PrivSSH, ssh_host.MachineAddress, ssh_host.SSHPort, ssh_host.MachineAddress))
+
+	opts := platform.MachineOptions{
+		MinMemory: 2048,
+	}
+
+	kdump_machine, err := c.NewMachineWithOptions(butane, opts)
+	if err != nil {
+		c.Fatalf("Unable to create test machine: %v", err)
+	}
+
+	// TODO : maybe add a retry loop +  timeout ?
+    time.Sleep(15 * time.Second)
+
+	// make sure kdump service is active
+	kdump_status := c.MustSSH(kdump_machine, "systemctl is-active kdump.service")
+	if string(kdump_status) == "inactive" {
+       c.Fatalf("Kdump.service is not ready: %s. Err: %v", string(kdump_status), err)
+	}
+
+	// crash the kernel
+	c.SSH(kdump_machine, "sudo su -c \"echo 'c' > /proc/sysrq-trigger\"")
```

Looking at the GitHub Actions error, one thing comes to my mind here:
if err != nil {
		c.Fatalf("Unable to execute: %v", err)
	}

---

This command will always hangs as it crashes the kernel so it will always errors out. 
Added a comment and explicitly ignored the error